scripts/services/getConfig.ts

```diff
@@ -24,5 +24,27 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
         "Must not contain newline or quote characters.",
     "copyable": true,
     "masked": true
-  }
+  },
+  "custom-electrs": {
+    "type": "object",
+    "name": "Custom Electrs (Emergency only)",
+    "description": "Switch to a custom Electrum server instance. This is only to be used in emergency situations where you have a position open but your embassy's Electrs instance has a problems.",
```

Comment:
I would remove the second sentence. The are several reasons why a user would want to have a different electrum server specified, "emergency" situations are just one of them. 

```suggestion
"description": "Switch to a custom Electrum server instance.",
```

Reply:
I reworded it!